libsupport/include/katana/EntityTypeManager.h

```diff
@@ -176,6 +191,8 @@ class KATANA_EXPORT EntityTypeManager {
   template <typename Container>
   Result<EntityTypeID> GetOrAddNonAtomicEntityTypeFromStrings(
       const Container& names) {
+    // An empty set is not a valid type name
```

Comment:
nit: I find this comment a little confusing. Why would the set be any kind of name?

Reply:
This is valid after #577. It should return `kUnknownEntityType`.

Reply:
Removed.